src/util/languageUtils.js

```diff
@@ -38,16 +38,10 @@ export const getLanguage = (
 	request,
 	supportedLangs,
 	defaultLang = LANG_DEFAULT
-) => {
-	// return the first language hit in the order of preference
-	supportedLangs.sort(l => l !== defaultLang);
```

Comment:
Looks like I just needed to review the `array.sort` API - this sorting function should be

```js
supportedLangs.sort(l => l === defaultLang ? -1 : 1);
```

As the deleted comment suggests, the `supportedLangs` are being sorted by their order of preference. The only preference we have is that the default language take precedence over all others, so the sort function is intended to guarantee that it appears at the top of the list rather than assuming that the list will be manually defined in order of preference.

You should get the correct behavior just by adjusting the sort function